diff --git a/backend/.env b/backend/.env
index 0f4322c..ca1299e 100644
--- a/backend/.env
+++ b/backend/.env
@@ -1,2 +1,4 @@
-MONGO_URL="mongodb://localhost:27017"
-DB_NAME="test_database"
\ No newline at end of file
+SUPABASE_URL=https://vsyrpfiwhjvahofxwytr.supabase.co
+SUPABASE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InZzeXJwZml3aGp2YWhvZnh3eXRyIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjAwMjYxNTgsImV4cCI6MjA3NTYwMjE1OH0.dO2c8YWIph2D95X7jFdlGYJ8MXyuyorkLcjQ6onH-HE
+MONGO_URL=mongodb://mongodb:27017
+DB_NAME=pezkuwi_mobile
\ No newline at end of file
